apps/src/netsim/NetSimLobby.js

```diff
@@ -215,86 +273,149 @@ NetSimLobby.prototype.refreshShardList_ = function () {
   });
 };
 
+NetSimLobby.prototype.useRandomShard = function () {
+  this.randomShardID = 'netsim_' + utils.createUuid();
+  this.useShard(this.randomShardID);
+};
+
+NetSimLobby.prototype.useShard = function (shardID) {
+  this.shardSelector_.empty();
+
+  $('<option>')
+      .val(shardID)
+      .html('My Private Network')
+      .appendTo(this.shardSelector_);
+
+  this.shardLinks_.
+      attr('href', this.buildShareLink(shardID)).
+      show();
```

Comment:
Interesting the "Indenting wrapped lines" perhaps implies that four spaces is correct (and I've probably been doing this "wrong" everywhere)